polaris/polaris/management/commands/watch_transactions.py

```diff
@@ -138,7 +144,7 @@ def process_response(cls, response, account):
         # transaction. This allows anchors to validate the actual amount sent in
         # execute_outgoing_transactions() and handle invalid amounts appropriately.
         transaction.amount_in = round(
-            Decimal(payment_op.amount), transaction.asset.significant_decimals,
+            Decimal(payment_op.amount.int64), transaction.asset.significant_decimals,
```

Comment:
@overcat is this correct? How do I get a string representation of the payment amount?